platform/Stream.cpp

```diff
@@ -112,14 +103,14 @@ void Stream::rewind() {
 }
 
 int Stream::isatty() {
-    return 0;
+    return 1;
```

Comment:
Hmm, seems fair for the fdopen change, but I'm not sure we should risk breaking backwards compatibility here. I'll leave it up to you all.